src/System.Net.Ping/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -10,7 +10,7 @@ namespace System.Net.NetworkInformation.Tests
     public class LoggingTest
     {
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "NetEventSource is only part of .NET Core")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework | TargetFrameworkMonikers.UapAot, "NetEventSource is only part of .NET Core and requires internal Reflection")]
```

Comment:
=> "NetEventSource is only part of .NET Core and requires internal Reflection.
          Does this not work because it need internal reflection?